src/shared/styles/components/top-bar.css

```
@@ -8,20 +8,19 @@
-
   display: flex;
   flex-direction: row;
-  gap: 1rem;
-
-  margin: 0;
-  padding: 0.5rem;
-
   align-items: center;
   justify-content: space-between;
 
+  margin: 0 auto;
```

Stylelint (17.7.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)